mozregression/main.py

```diff
@@ -178,25 +178,29 @@ def _bisect_inbounds(self, good_rev, bad_rev, ensure_good_and_bad=False,
                     return self._bisect_inbounds(good_rev, bad_rev,
                                                  expand=DEFAULT_EXPAND)
                 else:
+                    # This code is broken, it prints out the message even when
+                    # there are multiple bug numbers or commits in the range.
+                    # Somebody should fix it before re-enabling it.
+                    pass
```

Comment:
Could you change this to just 'return 0' and remove the changes below? I wouldn't make a big issue of it but the way you wrote the patch is causing travis to fail.